cmd-broken/machine-driver-google.go

```diff
@@ -0,0 +1,10 @@
+package main
```

Comment:
The one in here (cmd-broken) don't compile because the driver was not adapted properly.
They need to be fixed (not sure how - cli flags issues)